src/background/assessment-card-controller.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BaseStore } from 'common/base-store';
+import { AssessmentCardSelectionMessageCreator } from 'common/message-creators/assessment-card-selection-message-creator';
+import { AssessmentStoreData } from 'common/types/store-data/assessment-result-data';
+
+export class AssessmentCardController {
+    constructor(
```

Comment:
It doesn't look like this is ever getting constructed/ initialized yet. Is there any reason to put this off to another PR? Have you tested end to end with this being initialized?

Reply:
I'll add this